### PR TITLE
Elb v3 next resources

### PR DESCRIPTION
**Pull request by anton-sidelnikov**
- [x] pool
- [x] member
- [x] health_monitor
- [x] l7 policy
- [x] l7 rule
- [x] proxy unit tests

### CODE REVIEW

**Review by gtema — DISMISSED** [review]
Not really going very deep shortly before vacation.
Maybe you should look at approach with func tests in https://github.com/opentelekomcloud-infra/octavia-proxy/pull/36 (a very general setup/teardown for network). No problem with changing it in the followup

File: otcextensions/sdk/vlb/v3/_proxy.py [hunk 1]
```diff
@@ -343,10 +339,10 @@ def flavors(self, **query):
         )
 
     def get_flavor(self, flavor):
-        """Get a single instance
+        """Get a single flavor
 
-        :param instance: The value can be either the ID of an instance or a
-            :class:`~otcextensions.sdk.dds.v3.flavor.Flavor` instance.
+        :param flavor: The value can be either the ID of an flavor or a
+            :class:`~otcextensions.sdk.dds.v3.flavor.Flavor`.
```
Comment (lego963) [inline otcextensions/sdk/vlb/v3/_proxy.py]: dds flavor?

File: otcextensions/sdk/vlb/v3/member.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    #: Specifies the operating status of the backend server.
+    operating_status = resource.Body('operating_status')
+    #: Specifies the project ID.
+    project_id = resource.Body('project_id')
+    #: Specifies the port used by the backend server to receive requests.
+    protocol_port = resource.Body('protocol_port', type=int)
+    #: Specifies the ID of the subnet where the backend server works.
+    subnet_cidr_id = resource.Body('subnet_cidr_id')
+    #: Specifies the weight of the backend server.
+    weight = resource.Body('weight')
```
Comment (lego963) [inline otcextensions/sdk/vlb/v3/member.py]: should weight be `int`?

**Review by lego963 — CHANGES_REQUESTED** [review]

File: otcextensions/sdk/vlb/v3/member.py (hunk 2 above)
Comment (anton-sidelnikov) [reply otcextensions/sdk/vlb/v3/member.py]: fixed

File: otcextensions/sdk/vlb/v3/_proxy.py (hunk 1 above)
Comment (anton-sidelnikov) [reply otcextensions/sdk/vlb/v3/_proxy.py]: fixed

**Review by lego963 — APPROVED** [review]

**Pull request merged**